scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in

```diff
@@ -2813,6 +2813,23 @@ our @options = (
     category    => "dynamic",
   },
   {
+    name        =>  'ZM_TELEMETRY_INTERVAL',
+    default     =>  '14*24*60*60',
+    description =>  'Interval in seconds between telemetry updates.',
+    help        =>  'This value can be expressed as a mathematical expression for ease.',
+    type        =>  $types{string},
+    category    =>  'system',
+  },
+  {
+    name        => 'ZM_TELEMETRY_SERVER_ENDPOINT',
+    default     => 'https://zmanon:2b2d0b4skps@telemetry.zoneminder.com/zmtelemetry/testing5',
+    description => 'URL that ZoneMinder will send usage data to',
+    help        => qqq("
+        "),
+    type        => $types{url},
+    category    => 'system',
```

Comment:
Hmm.  I'm torn on this.  On one hand, it is nice to be able to update the value without modifying the perl script.  But that itself isn't really a pain point.

My real issue is with the 'system' category.  This isn't something that the user should be able to (easily) change.  Perhaps the 'hidden' category works better.  Makes it more easily configurable, but doesn't give the user easy access to update it.